### DISCUSSION TITLE
SSL with nginx and local network

### DISCUSSION THREAD

**Discussion by DerDominik** [dani-garcia/vaultwarden#2314] [Q&A] [answered]
Hi there,

I want to test a docker-vaultwarden inside of my local network (from outside only access via wireguard, otherwise no open port to the internet) and first steps have been done.
I configured the vaultwarden docker and a nginx-docker as reverse proxy (if I understand correctly), so I can use vaultwarden via my browser. But not I want to use a trused SSL connection from the client to the server.
With my Brave Browser I get the information that the connection is not safe (OS is Windows 10) and also the Bitwarden client cannot connect.

So is there a possibility to create a safe/trusted certificate, so I can connect without any safety problems to the server?
I tried some tutorials with Easy-RSA and OpenSSL, but nothing works.

Or is this only possible via LetsEncrypt? But then I also need a fixed domain?

Thanks for your help, **DerDominik**

**Comment by jjlin** [comment, marked as answer]
See https://github.com/dani-garcia/vaultwarden/wiki/Running-a-private-vaultwarden-instance-with-Let%27s-Encrypt-certs

**Reply by DerDominik** [reply]
Thanks, I had a little missunderstanding of the wiki page.
But after some searching and trying it is now running. 
Thanks!
